src/Data/StrMap.js

```diff
@@ -6,7 +6,7 @@
 exports._copy = function (m) {
   var r = {};
   for (var k in m) {
-    if (m.hasOwnProperty(k)) {
+    if ({}.hasOwnProperty.call(m, k)) {
```

Comment:
`var hasOwnProperty = {}.hasOwnProperty;` would also cache the lookup of the method. 👌🏻